org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/ResponseErrorCode.java

```diff
@@ -1,46 +0,0 @@
-/******************************************************************************
```

Comment:
I'm already removing this in #615.

Reply:
somehow my search in your pr did not find it :(